SJMaster/urls/__init__.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+from django.urls import path, include
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
+    path('', include("SJMaster.urls.student_urls"), name="student_urls"),
```

Comment:
Why not put each file inside its own app's package?

Reply:
Yea I guess that makes sense

Reply:
done